lib/web/mfa.go

```diff
@@ -211,3 +212,155 @@ func getDeviceUsage(reqUsage string) (proto.DeviceUsage, error) {
 
 	return deviceUsage, nil
 }
+
+type isMFARequiredDatabase struct {
+	// ServiceName is the database service name.
+	ServiceName string `json:"serviceName"`
+	// Protocol is the type of the database protocol.
+	Protocol string `json:"protocol"`
+	// Username is an optional database username.
+	Username string `json:"username,omitempty"`
+	// Name is an optional database name.
+	Name string `json:"name,omitempty"`
+}
+
+type isMFARequiredKube struct {
+	// Name is the name of the kube cluster.
+	Name string `json:"name"`
+}
+
+type isMFARequiredNode struct {
+	// Name can be node's hostname or UUID.
+	Name string `json:"name"`
```

Comment:
```suggestion
	NodeName string `json:"nodeName"`
```

Maybe Hostname or HostID would be better? I'm not sure.